cpp/src/groupby/sort/group_tdigest.cu

```diff
@@ -583,6 +688,7 @@ struct typed_group_tdigest {
                                   nearest_value_scalar_weights{},
                                   total_weight,
                                   cumulative_scalar_weight{group_offsets, group_labels},
+                                  col.null_count() > 0,
```

Comment:
I want to add a global `enum class column_nullability { has_nulls, no_null };` into `traits.hpp` then use it instead of using a boolean condition like this :smiley:

Reply:
:] I had a similar idea in my `lists::index_of()` [PR](https://github.com/rapidsai/cudf/pull/9510/files#diff-d66d386d623323d882af299672d1a96abf1c8ade91dcf2cf0ac529e3b45fb329R78).